language/de-DE/de-DE.com_content.ini

```diff
@@ -73,26 +66,28 @@ COM_CONTENT_NONE="Keine"
 COM_CONTENT_NUM_ITEMS_TIP="Beitragsanzahl"
 COM_CONTENT_NUM_ITEMS="Beitragsanzahl:"
 COM_CONTENT_ON_NEW_CONTENT="Ein neuer Beitrag mit dem Namen „%2$s“ wurde von „%1$s“ eingereicht."
-COM_CONTENT_ORDERING="Reihenfolge:<br />Neue Beiträge werden standardmäßig an erster Stelle in der Kategorie aufgelistet. Die Reihenfolge kann im Backend geändert werden."
+COM_CONTENT_ORDERING="Reihenfolge:<br>Neue Beiträge werden standardmäßig an erster Stelle in der Kategorie aufgelistet. Die Reihenfolge kann im Backend geändert werden."
 COM_CONTENT_PAGEBREAK_DOC_TITLE="Seitenumbruch"
 COM_CONTENT_PAGEBREAK_INSERT_BUTTON="Seitenumbruch einfügen"
 COM_CONTENT_PAGEBREAK_TITLE="Seitentitel:"
 COM_CONTENT_PAGEBREAK_TOC="Alias für Inhaltstabelle:"
 COM_CONTENT_PARENT="Hauptkategorie: %s"
-COM_CONTENT_PUBLISHED_DATE_ON="Veröffentlicht: %s"
 COM_CONTENT_PUBLISHED_DATE="Veröffentlichungsdatum"
+COM_CONTENT_PUBLISHED_DATE_ON="Veröffentlicht: %s"
 COM_CONTENT_PUBLISHING="Veröffentlichen"
-COM_CONTENT_RATINGS_COUNT="Bewertungen: %s"
 COM_CONTENT_RATINGS="Bewertung"
-COM_CONTENT_READ_MORE_TITLE="Weiterlesen ..."
+COM_CONTENT_RATINGS_COUNT="Bewertungen: %s"
 COM_CONTENT_READ_MORE="Weiterlesen: "
+COM_CONTENT_READ_MORE_TITLE="Weiterlesen ..."
 COM_CONTENT_REGISTER_TO_READ_MORE="Anmelden, um den ganzen Beitrag zu lesen"
 COM_CONTENT_RIGHT="Rechts"
 COM_CONTENT_SAVE_SUCCESS="Der Beitrag wurde gespeichert!"
 COM_CONTENT_SAVE_WARNING="Der gewählte Alias existiert bereits und wurde daher um eine Zahl ergänzt. Wenn der Alias geändert werden soll, bitte den Administrator der Website kontaktieren."
 COM_CONTENT_SELECT_AN_ARTICLE="Einen Beitrag auswählen"
+COM_CONTENT_STATE="Status"
 COM_CONTENT_SUBMIT_SAVE_SUCCESS="Der Beitrag wurde eingereicht."
 COM_CONTENT_TITLE_FILTER_LABEL="Titelfilter"
-COM_CONTENT_VOTES_COUNT="Abstimmungen: %s"
+COM_CONTENT_TRANSITION="Status"
```

Comment:
@zero-24 no match in joomla-cms?
https://github.com/joomla/joomla-cms/search?q=COM_CONTENT_TRANSITION&unscoped_q=COM_CONTENT_TRANSITION ?